pkg/ddevapp/ddevapp.go

```diff
@@ -46,87 +46,106 @@ const SiteConfigMissing = ".ddev/config.yaml missing"
 // SiteStopped defines the string used to denote when a site is in the stopped state.
 const SiteStopped = "stopped"
 
-// DdevApp is the struct that represents a ddev app
+// DdevApp is the struct that represents a ddev app, mostly its config
+// from config.yaml.
 type DdevApp struct {
-	AppConfig *Config
+	APIVersion            string               `yaml:"APIVersion"`
+	Name                  string               `yaml:"name"`
+	AppType               string               `yaml:"type"`
+	Docroot               string               `yaml:"docroot"`
+	WebImage              string               `yaml:"webimage"`
+	DBImage               string               `yaml:"dbimage"`
+	DBAImage              string               `yaml:"dbaimage"`
+	ConfigPath            string               `yaml:"-"`
+	AppRoot               string               `yaml:"-"`
```

Comment:
this one stutters too, but not as obvious an alternative. maybe RootDir? ProjectDir? I could also just live with this one continuing to stutter if we don't find a better fit.

Reply:
Yeah. I'm open to changing but "Root" seems ambiguous alone.